### DISCUSSION TITLE
How to use surround  in Visual-block mode

### DISCUSSION THREAD

**Discussion by Jagddish** [kylechui/nvim-surround#348] [Q&A] [answered]
after selecting these 3 lines in visual-block mode, when I tried to use St it would only wrap around the letter
same result with S<

          before
      <li>*One</li>
      <li>*two</li>
      <li>*three</li>
      
           after
      <li><a>O</a>ne</li>
      <li><a>t</a>wo</li>
      <li><a>t</a>hree</li>

**Comment by kylechui** [comment, marked as answer]
Are you selecting the whole lines or just the characters? Try something like `0<C-v>_St` (go to beginning of line, start visual block, go to end of line, surround).

EDIT: Just realized the words inside aren't uniform length; best way might be to just do one row and dot-repeat the rest, or record a macro.

**Reply by Jagddish** [reply]
Thank you
